### DISCUSSION TITLE
How does unpin work?

### DISCUSSION THREAD

**Discussion by madprops** [Taiko2k/Tauon#1200] [General]
I noticed my pins at the tab bar, at the top, get unpinned by themselves automatically or in a way I don't understand. I don't think I'm accidentally clicking Unpin or middle clicking them. But some actions make them unpin. The playlists are not removed as they still reside in the left panel.

How does the automatic unpin work, and, is there a way to disable it? I'd prefer to unpin them manually through middle click or context menu.

**Comment by Taiko2k** [comment]
They can unpin if you drag them off

**Comment by madprops** [comment]
Strange. I'm trying to force that by dragging or semi draggin them. But it won't happen. But other times by simply clicking the tracklist they get unpinned by themselves. I'll keep testing this.

**Reply by madprops** [reply]
I guess it might be indeed an accidental dragging action.

**Comment by madprops** [comment]
Haha, I'm almost sure there must be a bug happening because all my pins keep getting unpinned. I doubt it's me dragging them so sloppily.  But I'm not sure how to reproduce it, but if I find how I'll let you know.

**Comment by madprops** [comment]
https://github.com/Taiko2k/Tauon/issues/1203
